github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -193,7 +193,7 @@ presubmits:
           resources:
             requests:
               memory: "29Gi"
-  - name: pull-kubevirt-e2e-k8s-multus-1.13.3
+  - name: pull-kubevirt-e2e-k8s-cnao-1.17
```

Comment:
Could you please link a PR where you test this branch of kubevirtci in kubevirt?

Reply:
Added to description